scrapy/settings/default_settings.py

```diff
@@ -250,6 +250,7 @@
 SCHEDULER_PRIORITY_QUEUE = 'queuelib.PriorityQueue'
 
 SPIDER_LOADER_CLASS = 'scrapy.spiderloader.SpiderLoader'
+SPIDER_LOADER_WARN_ONLY = False
```

Comment:
Could you please add docs for this option? Or are you waiting for @vshlapakov's feedback?